tests/rptest/tests/pandaproxy_test.py

```diff
@@ -147,10 +147,7 @@ def _create_topics(self,
                 TopicSpec(name=name,
                           partition_count=partitions,
                           replication_factor=replicas))
-        wait_until(lambda: set(names).issubset(self._get_topics().json()),
-                   timeout_sec=30,
-                   backoff_sec=1,
-                   err_msg="Topics failed to settle")
+        assert set(names).issubset(self._get_topics().json())
```

Comment:
any particular reason for this? using wait_until solves the issue of transit issues and shouldn't add any overhead if the first call to the test succeeds.